guides/taxation/overview-of-taxation.md

```diff
@@ -0,0 +1,149 @@
+# Overview of taxation
+
+Solidus uses the `Spree::Adjustment` model to apply taxes to orders. This way,
+there can be multiple tax adjustments applied to each order.
+
+Using adjustments helps account for some of the complexities of tax, especially
+if a store sells internationally:
+
+- Orders may include products with different tax categories or rates.
+- Shipments may require special calculations if you are shipping to or from
+  locations where there are specific taxation rules for shipments.
+- Taxes may be included in a product's price or not in included in a product's
+  price depending on a country's taxation rules.
+
+<!-- TODO:
+  - Add links to the locations guide and specifically the [zones](#) article.
+  - Add links to the adjustments guide.
+-->
+
+Solidus uses the `Spree::TaxCategory` and `Spree::TaxRate` models to represent
+taxes for each line item in an order. See the sections below for more
+information about how these models work. 
+
+See the [Order taxation](#order-taxation) section for an overview of the order
+taxation process.
+
+## Tax categories
+
+Tax categories can be used to ensure particular products are taxed as required.
+For example, if your business is based in Minnesota, you need to charge tax on
+tech products but do not need to charge tax on clothing. You could set up two
+tax categories, **Clothing** and **Tech** which you would apply to products of
+either type.
+
+By default, new products do not have a set tax category. Administrators can set
+the tax category while creating the product or set it later while editing the
+product. 
+
+## Tax rates
+
+Tax rates define the amount of tax that should be charged on an item. You might
```

Comment:
...on items. Using plural here I think makes it more clear that it's a rule/configuration, not a particular entity.